app/views/options/index.html.haml

```diff
@@ -1,3 +1,14 @@
+:javascript
+  // Hide default donation type radio group unless recurring donations are allowed
+  function showDefaultDonation(select_allow_recurring_donations) {
```

Comment:
why is this done with javascript?  don't we already know at rendering time whether recurring donations are allowed, based on the option value?  this should be statically part of the rendered view, conditioned on that option.  javascript should only be used when there isn't a graceful way to do something at the time the response is first served. also, even if using javascript, since jquery is in use it is better to specify behavior, as in `$(element).hide()`, than to set attributes, as in `$(element).style.visiblity = 'hidden'`.  or even attach or remove css class(es) from an element.  but in general you shoudl NEVER directly modify specific style attributes inline, nor using hardwired constant values; this takes all control away from the presentatoin layer.  in general i think this functionality can be done entirely with JS unless i am missing something.